server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/DeleteAction.java

```diff
@@ -91,34 +89,36 @@ public void handle(Request request, Response response) throws Exception {
     }
   }
 
-  private void checkNotTryingToDeleteDefaultGroup(DbSession dbSession, long groupId) {
+  private void checkNotTryingToDeleteDefaultGroup(DbSession dbSession, GroupId group) {
+    // TODO support organizations
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)